src/sentry/static/sentry/app/views/events/utils/eventsRequest.tsx

```diff
@@ -224,25 +262,19 @@ class EventsRequest extends React.PureComponent {
       ),
     };
   };
-
   /**
    * Aggregate all counts for each time stamp
    */
-  transformAggregatedTimeseries = (data, seriesName) => {
-    if (!data) {
-      return null;
-    }
-
+  transformAggregatedTimeseries = (data: EventsStatsData, seriesName: string): Series => {
     return {
       seriesName,
       data: this.calculateTotalsPerTimestamp(data),
     };
   };
-
   /**
    * Transforms query response into timeseries data to be used in a chart
    */
-  transformTimeseriesData = data => {
+  transformTimeseriesData = (data: EventsStatsData): Series[] => {
```

Comment:
Since it only ever returns a single series should we do `[Series]` instead of `Series[]`?